.github/workflows/build_rpms.yml

```diff
@@ -0,0 +1,39 @@
+name: RPM Build
+
+on: [pull_request, push]
+
+jobs:
+  rpm_build:
+    name: "Build RPMs"
+    runs-on: ubuntu-latest
+    container: registry.gitlab.com/majorhayden/containers/rpm-builder:latest
+    steps:
+      - name: Prepare container
+        run: |
+          echo "fastestmirror=1" >> /etc/dnf/dnf.conf
+          echo "install_weak_deps=0" >> /etc/dnf/dnf.conf
+          dnf -y upgrade
+          dnf -y install dnf-plugins-core git make rpm-build
+          mkdir rpms
+
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          path: osbuild
+
+      - name: Install RPM build dependencies
+        run: dnf -y builddep osbuild/osbuild.spec
+
+      - name: Build RPMs
+        run: |
+          pushd osbuild
+            make rpm
+          popd
+          cp -av osbuild/output/*/*.rpm rpms/
+          date > rpms/timestamp.txt
+
+      - name: "Upload artifacts"
+        uses: actions/upload-artifact@v1
+        with:
+          name: Packages
+          path: rpms/
```

Comment:
Cool! Will we keep these paths stable so other scripts can fetch them? I have not looked into the github-actions artifact API, but we should be able to fetch them based on a commit-sha1, right?

Reply:
We should be able to, but I have not gotten that far yet. ;)